src/components/App.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import Note from "./Note";
+import CreateArea from "./CreateArea";
+
+function App() {
+  // the useState hook for updating the list of notes on UI
+  // useState() takes in initial state as parameter
+  // returns an array containing the state and a setState function
+  const [list, setList] = useState([]);
```

Comment:
I don't think "list" is the best name for the notes list. When I read list, I think of the data structure and not of the notes list. Also, the data structure is an array, so it's a bit confusing.

A better name would be "notes", no?